testing/child-subnet-infra/README.md

```diff
@@ -0,0 +1,14 @@
+# Child Subnet Infra 
```

Comment:
Depending on how you are planning to run the tests this README should explain how to run the test cases you are currently working on for the integration tests.